appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/visitor/OpenApiWalker.java

```diff
@@ -133,8 +133,8 @@ public OpenApiWalker(OpenAPI api, Set<Class<?>> allowedClasses, Map<String, Set<
     @Override
     public void accept(ApiVisitor visitor) {
```

Comment:
FYI: I change the `processAnnotations` to always only processing one annotation at a time as processing two (+"alt") was quite confusing to understand and how to change it to also handle "inherited" annotations. The alternative(s) are now stated as last argument so they can be taking into account when deciding if annotation on method level should inherit from class level. This now is their only function so this semantic now becomes more apparent.